### PR TITLE
Some changes to clarify what Freenet is and how to download & install

### PR DESCRIPTION
**Pull request by gckrol**
After looking at the site again after a while, I noticed that it wasn't introducing Freenet (and the installation process) well enough. I've added some text to clarify.

### CODE REVIEW

File: pages/download.py [hunk 1]
```diff
@@ -87,13 +87,26 @@ def __init__(self):
         self.title = _("Download")
         self.slug = "download"
     def get_content(self):
+        download_intro = md(_("""
+To access Freenet, you first need to install the main application. Freenet will run in the background and you can use
+your browser to change settings and access content. There are other applications that you can install
+at a later time to add more functionality.
```
Comment (bertm) [inline pages/download.py]: Please avoid breaking/wrapping lines at arbitrary points. Doing so introduces unnecessary complexity in the Git history and might make use of translation memories slightly harder.

File: pages/download.py [hunk 2]
```diff
@@ -299,6 +312,10 @@ def get_content(self):
 """) + "\n\n" + """
 [url_addfriend]: http://127.0.0.1:8888/addfriend/
 [url_seednode_info]: https://wiki.freenetproject.org/Seed_nodes#Seed_node
+""") + "\n\n" + _("""
+You don't have to add friends right now. Freenet will automatically connect to strangers
+and will work just fine. However, your (or someone elses) government may be able to find out who you are with
```
Comment (bertm) [inline pages/download.py]: Without friends, Freenet will only work "just fine" if low or medium security is chosen.

File: pages/download.py [hunk 3]
```diff
@@ -299,6 +312,10 @@ def get_content(self):
 """) + "\n\n" + """
 [url_addfriend]: http://127.0.0.1:8888/addfriend/
 [url_seednode_info]: https://wiki.freenetproject.org/Seed_nodes#Seed_node
+""") + "\n\n" + _("""
+You don't have to add friends right now. Freenet will automatically connect to strangers
```
Comment (bertm) [inline pages/download.py]: While probably not absolutely necessary, please keep the above URL index paragraph as the final paragraph in the markdown.

File: pages/download.py [hunk 4]
```diff
@@ -87,13 +87,26 @@ def __init__(self):
...
+        download_intro = md(_("""
+To access Freenet, you first need to install the main application. Freenet will run in the background and you can use
+your browser to change settings and access content. There are other applications that you can install
+at a later time to add more functionality.
+
+Steps:
+
+* [Download and install Freenet](#autostart)
+* [Add friends (or connect to strangers)](#note)
+* [Start using Freenet!](#usage)
```
Comment (bertm) [inline pages/download.py]: URLs may change in the future and should therefore not be hardcoded in the translation unit. Please use [reference-style link syntax](http://daringfireball.net/projects/markdown/syntax#link) `[link text][link_identifier]` and append an index paragraph with the URL identifiers.

File: pages/download.py [hunk 5]
```diff
@@ -87,13 +87,26 @@ def __init__(self):
...
-        return show_hide_script()+text("""<a id="autostart"></a>""" + _("Show instructions for:")+"""
+        return show_hide_script()+text(download_intro)+text("""<a id="autostart"></a><p>""" + _("Show installation instructions for:")+"""
       <a href="javascript:showDiv('windows');hideDiv('macos');hideDiv('unix');">Windows
         </a>, <a href="javascript:hideDiv('windows');showDiv('macos');hideDiv('unix');">OS X
         </a>, <a href="javascript:hideDiv('windows');hideDiv('macos');showDiv('unix');">GNU/Linux & POSIX
-        </a><br>
-"""+div("windows",md(_("""
+        </a></p>
+"""+md(_("""
+*Freenet is Open Source (GPLv2+), get the source code on [GitHub](https://github.com/freenet/fred).*
```
Comment (bertm) [inline pages/download.py]: Same here, please use [reference-style links](http://daringfireball.net/projects/markdown/syntax#link).

File: pages/index.py [hunk 6]
```diff
@@ -155,6 +162,8 @@ def slider_item(title, text, active=""):
         nonprofit = _("The Freenet Project Inc is a non-profit 501(c)(3) organization.")
         tax_deductable = _("""Donations are tax-deductible.""")
         return substitute_html(content,
+            html__slogan=slogan,
+            html__mission=mission,
```
Comment (bertm) [inline pages/index.py]: This is not arbitrary HTML but rather markdown content. Although this approach is valid (HTML is a superclass of Markdown), please use the `md__` prefix instead: it lets the editor of the template know that textual content is expected at this position.

**Comment by gckrol** [comment]
@bertm Thanks for your feedback, I'll keep that in mind. I'm not sure if I feel like amending this pull request right now though.

File: pages/download.py (hunk 2 above)
Comment (Thynix) [inline pages/download.py]: Yup. Also medium is called "normal" in the UI.

**Comment by gckrol** [comment]
Ok, should be fine now. I've added a short bit of text to the "What is Freenet?" section on the main page as well.

**Comment by Thynix** [comment]
Thanks!

**Pull request merged**